superchain/configs/sepolia-dev-0/superchain.yaml

```
@@ -0,0 +1,8 @@
+name: Sepolia Dev 0
+l1:
+  chain_id: 11155111
+  public_rpc: https://ethereum-sepolia-rpc.allthatnode.com
+  explorer: https://sepolia.etherscan.io
+
+protocol_versions_addr: "0x252CbE9517F731C618961D890D534183822dcC8d"
+superchain_config_addr: "0x02d91Cf852423640d93920BE0CAdceC0E7A00FA7"
```

You may want to add two entries
```suggestion
superchain_config_addr: "0x02d91Cf852423640d93920BE0CAdceC0E7A00FA7"

canyon_time: xxx # Tue Nov xx 17:00:00 UTC 2023
delta_time: yyy  # Fri Dec yy 00:00:00 UTC 2023
```
that refer to the activation times on `base-devnet-0`. This way, nodes syncing using the superchain get the correct activation times from here.

We will add the ecotone activation time later today in a separate pr, after we've agreed on a time at standup.

---

We agreed to put both Base's activation times here, and add a delta-at-genesis override into op-node.

---

Set the base override times. Thanks